src/platform/src/Bridge/Ollama/Ollama.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Bridge\Ollama;
+
+use Symfony\AI\Platform\Capability;
+use Symfony\AI\Platform\Model;
+
+/**
+ * @author Joshua Behrens <code@joshua-behrens.de>
+ */
+class Ollama extends Model
+{
+    public const DEEPSEEK_R_1 = 'deepseek-r1'; // TOOL
+    public const GEMMA_3_N = 'gemma3n';
+    public const GEMMA_3 = 'gemma3';
+    public const QWEN_3 = 'qwen3'; // TOOL
+    public const QWEN_2_5_VL = 'qwen2.5vl';
+    public const LLAMA_3_1 = 'llama3.1'; // TOOL
+    public const LLAMA_3_2 = 'llama3.2'; // TOOL
+    public const MISTRAL = 'mistral'; // TOOL
+    public const QWEN_2_5 = 'qwen2.5'; // TOOL
+    public const LLAMA_3 = 'llama3';
+    public const LLAVA = 'llava';
+    public const PHI_3 = 'phi3';
+    public const GEMMA_2 = 'gemma2';
+    public const QWEN_2_5_CODER = 'qwen2.5-coder'; // TOOL
+    public const GEMMA = 'gemma';
+    public const QWEN = 'qwen';
+    public const QWEN_2 = 'qwen2'; // TOOL
+    public const LLAMA_2 = 'llama2';
```

Comment:
I think we can remove the `// TOOL` comments, once we have a unit test for the regex, using a data provider